modules/xfem/doc/content/modules/xfem/index.md

```diff
@@ -5,8 +5,8 @@ be used in conjuction with any number of other physics modules to enrich the sol
 differential equations (PDEs) with some form of discontinuity. XFEM theory and algorithms employed
 throughout this module are detailed in the following links:
 
-- [Theory Manual](xfem/theory.md)
-- [Algorithms](xfem/algorithms.md)
+- [Theory Manual](xfem/theory/theory.md)
```

Comment:
You had several bad paths, just look at the error message. It says as much and give you suggestions:

```
MooseDocs.base.renderers (Process-5): 
RENDER ERROR: Unable to locate a page that ends with the name 'xfem/theory.md'. Did you mean one of the following:
     theory.md
     theory/theory.md
     xfem_srs.md
     xfem/index.md
     xfem_vvr.md

/Users/slauae/projects/moose/modules/xfem/doc/content/modules/xfem/index.md:6
 ┌─────────────────────────────────────────────────────────────────────────────────────────────────┐
6│[Theory Manual](xfem/theory.md)                                                                  │
 └─────────────────────────────────────────────────────────────────────────────────────────────────┘
Traceback (most recent call last):
  File "/Users/slauae/projects/moose/python/MooseDocs/base/renderers.py", line 85, in render
    el = func(parent, token, page) if func else parent
  File "/Users/slauae/projects/moose/python/MooseDocs/extensions/autolink.py", line 225, in createHTML
    warn_on_zero=token['warning'])
  File "/Users/slauae/projects/moose/python/MooseDocs/base/Translator.py", line 191, in findPage
    raise exceptions.MooseDocsException(msg)
MooseDocs.common.exceptions.MooseDocsException: Unable to locate a page that ends with the name 'xfem/theory.md'. Did you mean one of the following:
     theory.md
     theory/theory.md
     xfem_srs.md
     xfem/index.md
     xfem_vvr.md

```